helper_base.php

```diff
@@ -765,7 +765,7 @@ public static function get_resources()
       self::$resource_list = array (
         'indiciaFns' => array('deps' =>array('jquery'), 'javascript' => array(self::$js_path."indicia.functions.js")),
         'jquery' => array('javascript' => array(self::$js_path."jquery.js",self::$js_path."ie_vml_sizzlepatch_2.js")),
-        'openlayers' => array('javascript' => array(self::$js_path.(function_exists('iform_openlayers_get_file') ? iform_openlayers_get_file() : "OpenLayers.js"),
+        'openlayers' => array('javascript' => array(self::$js_path.(function_exists('iform_openlayers_get_file') ? iform_openlayers_get_file() : "OpenLayers-min.js"),
```

Comment:
Can this switch to the -min version of the file be avoided, by converting iform_openlayers to D8? I'm just slightly concerned that the file is bigger since the OpenLayers.js file has been optimised for Indicia. Or can we rebuild the file we use? Not sure the latter will be easy though. Perhaps we could manually edit OpenLayers.js to add code for the missing class from the minified file? Just a few thoughts - if none of these options are easy then the increase in file size might be something we just have to accept.

Reply:
I have implemented the iform_openlayers module for d8